doc/api/modules.md

```diff
@@ -859,3 +874,4 @@ const builtin = require('module').builtinModules;
 [module resolution]: #modules_all_together
 [module wrapper]: #modules_the_module_wrapper
 [native addons]: addons.html
+[`require`]: #modules_require
```

Comment:
Maybe this better be placed after the `` [`path.dirname()`] `` reference, ASCII-wise.